README.md

```diff
@@ -43,69 +43,95 @@ remotes::install_github("mrc-ide/frogger", upgrade = FALSE)
-|---------------|---------------------------|--------------------------------------------------------|
-| NG            | num_genders               | Number of genders                                      |
-| pAG           | age_groups_pop            | Number of age groups in population, 81 for 0 to 80+    |
-| pIDX_FERT     | fertility_first_age_group | First age group index eligible for fertility           |
-| pAG_FERT      | age_groups_fertiity       | Number of ages eligible for fertility                  |
-| pIDX_HIVADULT | hiv_adult_first_age_group | Index of the first age group to be considered an adult |
-| hAG           | age_groups_hiv            | Number of age groups in HIV population                 |
-| hDS           | disease_stages            | Number of disease stages                               |
-| hTS           | treatment_stages          | Number of treatment stages                             |
-
-### Model settings
-
-| Leapfrog           | Frogger             | Details                           |
-|--------------------|---------------------|-----------------------------------|
-| sim_years          | sim_years           | Number of simulation years to run |
-| hiv_steps_per_year | hiv_steps_per_year  |                                   |
-| t_ART_start        | time_art_start      | Time step to start modelling ART  |
-| hAG_SPAN           | hiv_age_groups_span | Array of HIV age group sizes      |
+Frogger naming rules
+
+* p prefix indicates this is age stratified by single year i.e. same age stratification as total population
+* h prefix indicates this is age stratified by same age stratification as HIV population, this might be single year from
+  15+ or could be coarse age groups
+* idx prefix means this is an index
+* Uppercase means this is a dimension of the state space e.g. `NS` or `pAG`
+
+### State space
+
+In `StateSpace` struct
+
+| Leapfrog           | Frogger            | Details                                             |
+|--------------------|--------------------|-----------------------------------------------------|
+| NG                 | NS                 | Number of sexes                                     |
+| pAG                | pAG                | Number of age groups in population, 81 for 0 to 80+ |
+| hAG                | hAG                | Number of age groups in HIV population              |
+| hDS                | hDS                | Number of disease stages                            |
+| hTS                | hTS                | Number of treatment stages                          |
+| sim_years          | sim_years          | Number of simulation years to run                   |
+| hiv_steps_per_year | hiv_steps_per_year | Number of HIV model time steps per year             |
+
+#### Loop variable convention
+
+* `NS` is `s`
+* `pAG` is `a`
+* `hAG` is `ha`
+* `hDS` is `hm`
+* `hTS` is `hu`
```

Comment:
Actually why is `hDS` `hm` and `hTS` `hu`? 

I think I'd fine something like `hds` and `hts` easier to read, or `ds` and `ts`?

Reply:
I think `m` and `u` were fairly arbitrary.  I adopted those for the loops in the main population arrays (perhaps they were in earlier `epp` code)  and affixed the `h` when looping in these arrays.

Open to change. But `hts` is problematic because we use that elsewhere for "HIV time step" (loop over the 0.1 year steps)

Reply:
Let's try `hd` and `ht` then? Consistent with `ha` etc. but has an initial a bit more meaningful initial I think